cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxEditBoxHelper.java

```diff
@@ -102,16 +102,6 @@ public void run() {
                 editBox.setTextColor(Color.WHITE);
                 editBox.setSingleLine();
                 editBox.setOpenGLViewScaleX(scaleX);
-                Resources r = mCocos2dxActivity.getResources();
```

Comment:
Now I am not using any default padding, since if Api is available for padding then User can set padding what he needed. So we don't need to set default padding.